protocol/syncer_test.go

```diff
@@ -284,7 +283,7 @@ func TestWatchSyncWithPeer(t *testing.T) {
 			go func() {
 				syncer.WatchSyncWithPeer(peer, func(b *types.Block) bool {
 					// sync until latest block
-					return b.Header.Number < latestBlock.Header.Number
+					return !(b.Header.Number < latestBlock.Header.Number)
```

Comment:
@munna0908 please flip the `<`, no reason to `!` the entire expression, very confusing that way

Reply:
@lazartravica fixed it [link](https://github.com/0xPolygon/polygon-sdk/pull/205/commits/07d18eb1a029eb68952a217913493ce537ee954c)